src/codegate/api/v1.py

```diff
@@ -665,7 +757,7 @@ async def get_workspace_token_usage(workspace_name: str) -> v1_models.TokenUsage
         raise HTTPException(status_code=500, detail="Internal server error")
 
     try:
-        prompts_outputs = await dbreader.get_prompts_with_output(ws.id)
+        prompts_outputs = await dbreader.get_prompts_with_output(worskpace_id=ws.id)
```

Comment:
small typo was causing this endpoint to 500
```suggestion
        prompts_outputs = await dbreader.get_prompts_with_output(workspace_id=ws.id)
```